cafe/db/schema.rb

```diff
@@ -0,0 +1,82 @@
+# encoding: UTF-8
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20221003092219) do
+
+  create_table "carts", force: :cascade do |t|
+    t.string   "menu_category_name"
+    t.integer  "menu_item_id"
+    t.string   "menu_item_name"
+    t.decimal  "menu_item_price"
+    t.datetime "created_at",         null: false
+    t.datetime "updated_at",         null: false
+    t.integer  "cart_id"
+    t.integer  "user_id"
+  end
+
+  add_index "carts", ["cart_id"], name: "index_carts_on_cart_id"
+
+  create_table "menu_categories", force: :cascade do |t|
+    t.integer  "menuid"
+    t.string   "name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "menu_items", force: :cascade do |t|
+    t.integer  "menu_category_id"
+    t.string   "name"
+    t.text     "description"
+    t.decimal  "price"
+    t.datetime "created_at",                   null: false
+    t.datetime "updated_at",                   null: false
+    t.integer  "sales_count",      default: 0, null: false
```

Comment:
items shouldn't have sales info. Sales should be derived from orders. Shouldn' have it here.
Will discuss more on this.

Reply:
Can be derived from the numbers of the order made for the item.